src/tests/ftest/pool/destroy_rebuild.py

```diff
@@ -58,5 +55,5 @@ def test_destroy_while_rebuilding(self):
         self.pool.destroy()
 
         self.log.info("Test Passed")
-        self.get_dmg_command().system_start(",".join(ranks))
-        self.server_managers[0].update_expected_states(",".join(ranks), ["joined"])
+        self.get_dmg_command().system_start(",".join([str(x) for x in ranks]))
```

Comment:
We have a general_utils method for this:
```suggestion
        self.get_dmg_command().system_start(list_to_str(value=ranks))
```